### DISCUSSION TITLE
Welcome to curriculum Discussions!

### DISCUSSION THREAD

**Discussion by wagslane** [bootdotdev/curriculum#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wagslane** [comment]
Because we're a site that focuses on teaching backend and computer science,
not frontend.

**Comment by And-Mes** [comment]
will there be a Java course? im studying java in my class right now and would like to get extra practice with it.

**Reply by Atomk** [reply]
I get that Java is really popular and I've used it and I don't want anything to do with it or anything related to Oracle

**Comment by hassanein1130-dev** [comment]
im hoping you can get some languages that are really iportant such as java c# and c++

**Comment by anand-anshul** [comment]
i would love to see more courses on CS path. Since i dont have a CS degree i always feel lacking in knowledge related to core CS. engineering major BTW

**Comment by mehmetcagriekici** [comment]
Hello,

I am Mehmet Çağrı Ekici, I've been using Boot.dev since July 2025 and I am currently level 126. 
https://www.boot.dev/u/callsower

I wanted to suggest that you could add an uncompleted/completed switch/button to the courses catalog. 

Thank you.

**Comment by mohammad-rizwan-hussain** [comment]
Hey @wagslane 
I've been eagerly waiting for CS courses like Networking and Distributed Systems for a long time, and I see they're WIP. I also noticed the recent "very WIP" AI course commits.

One of the main reasons I chose Boot.dev was to build strong foundations(Building Blocks). Would it be possible to prioritize and release the CS courses before the AI courses?
